// @esri/calcite-components/dist/cjs/pt-br-cad4fd70.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v1.5.0
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-30d9d752.js');
require('./index-9f8ce657.js');
require('./form-c8a8a683.js');
require('./dom-b7601405.js');
require('./guid-1a635f47.js');
require('./resources-53a07e71.js');
require('./interactive-793314f0.js');
require('./key-e590036d.js');
require('./label-22e5c734.js');
require('./loadable-d46f87a1.js');
require('./locale-7c3cc149.js');
require('./observers-e29e2712.js');
require('./focusTrapComponent-7a217706.js');
require('./t9n-8444b9f7.js');

// Portuguese (Brazil) [pt-br]
var locale = {
  name: 'pt-br',
  weekdays: 'domingo_segunda-feira_terça-feira_quarta-feira_quinta-feira_sexta-feira_sábado'.split('_'),
  weekdaysShort: 'dom_seg_ter_qua_qui_sex_sáb'.split('_'),
  weekdaysMin: 'Do_2ª_3ª_4ª_5ª_6ª_Sá'.split('_'),
  months: 'janeiro_fevereiro_março_abril_maio_junho_julho_agosto_setembro_outubro_novembro_dezembro'.split('_'),
  monthsShort: 'jan_fev_mar_abr_mai_jun_jul_ago_set_out_nov_dez'.split('_'),
  ordinal: function ordinal(n) {
    return n + "\xBA";
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD/MM/YYYY',
    LL: 'D [de] MMMM [de] YYYY',
    LLL: 'D [de] MMMM [de] YYYY [às] HH:mm',
    LLLL: 'dddd, D [de] MMMM [de] YYYY [às] HH:mm'
  },
  relativeTime: {
    future: 'em %s',
    past: 'há %s',
    s: 'poucos segundos',
    m: 'um minuto',
    mm: '%d minutos',
    h: 'uma hora',
    hh: '%d horas',
    d: 'um dia',
    dd: '%d dias',
    M: 'um mês',
    MM: '%d meses',
    y: 'um ano',
    yy: '%d anos'
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports.default = locale;
